src/EuropaSearch/EuropaSearchConfig.php

```diff
@@ -179,9 +209,73 @@ public function setUserPassword($userPassword)
     }
 
     /**
-     * {@inheritDoc}
+     * Gets the connection configuration for the Indexing and Search services.
+     *
+     * @return array
```

Comment:
If I'm not mistaken this description correspondes to the one done in line 63, can't we poing there (with a  @see tag) so that we avoid repeating such a long comment?